src/main/charts/confluence/templates/ingress-setup.yaml

```diff
@@ -29,14 +29,14 @@ spec:
     - host: {{ .Values.ingress.host }}
       http:
         paths:
-          - path: "{{ trimSuffix "/" .Values.ingress.path }}/setup"
+          - path: "{{ include "confluence.ingressPath" . }}/setup"
```

Comment:
If `Values.ingress.path` and `.Values.confluence.service.contextPath`, this will result to `//setup` URL and the ingress will be broken

Reply:
Actually it wound be awesome to have the unit test which is going to assert the exact path of Confluence ingresses. This way we will never be able to have something like `//setup` again

Reply:
wow, we see at the same time, I just uploaded a patch to fix this